UnitySDK/Assets/ML-Agents/Editor/DemonstrationImporter.cs

```diff
@@ -29,7 +29,7 @@ public override void OnImportAsset(AssetImportContext ctx)
                 Stream reader = File.OpenRead(ctx.assetPath);
 
                 var metaDataProto = DemonstrationMetaProto.Parser.ParseDelimitedFrom(reader);
-                var metaData = new DemonstrationMetaData(metaDataProto);
+                var metaData = metaDataProto.FromProto();
```

Comment:
The naming feels weird on this. Guess I'm just used to `Foo.FromBar()` be a static method in `Foo` not an instance method on `Bar`

Reply:
It's technically a static method, just looks like an instance method.

Reply:
But I agree with your sentiment.  I just wanted to remove as much Grpc code away from code that doesn't need to know about it